.pre-commit-config.yaml

```diff
@@ -5,14 +5,9 @@ default_stages:
     - commit
     - push
 minimum_pre_commit_version: 2.16.0
+ci:
+    skip: [mypy]
 repos:
-    - repo: https://github.com/nbQA-dev/nbQA
-      rev: 1.6.3
-      hooks:
-          - id: nbqa-pyupgrade
-            args: [--py38-plus]
-          - id: nbqa-black
-          - id: nbqa-isort
     - repo: https://github.com/psf/black
```

Comment:
would not remove this cause we will have notebooks here I think even if just few